durabletask/internal/shared.py

```diff
@@ -25,8 +23,10 @@ def get_default_host_address() -> str:
 
 def get_grpc_channel(
         host_address: Optional[str],
-        metadata: Optional[list[tuple[str, str]]],
-        secure_channel: bool = False) -> grpc.Channel:
+        metadata: Optional[list[tuple[str, str]]] = None,
+        secure_channel: bool = False,
+        interceptors: Optional[list] = None) -> grpc.Channel:
```

Comment:
Same question about whether we should be further constraining the allowed types inside of this `list`.